### PR TITLE
Add section about out F-Droid nightlies repository

### PR DESCRIPTION
**Pull request by pstorch**

### CODE REVIEW

**Comment by dennisguse** [comment]
Two tiny things:
* can we add the fingerprint into the QR code?
* (not related with this PR) should we also use the count of git commits as version code?
  [otherwise even no change aka commit leads to a new update]

**Comment by pstorch** [comment]
>  can we add the fingerprint into the QR code?

Hmm, yeah repo fingerprint. I saw it once somewhere, but didn't investigate yet. I guess I need to check a few things first. How to properly do it. At the moment the repo index is signed with an automatically generated key from the fdroid tool. Does it need to be the same as for signing the APKs? 


 >  (not related with this PR) should we also use the count of git commits as version code?

You mean the number of commits since the last nightly? I just figured out how to get the last 5 commit messages for the changelog. This I wanted to implement next.

**Comment by dennisguse** [comment]
I got the fingerprint from F-Droid (after adding the repo, you can select it in the settings; even shows a QR code): https://fdroid.storchp.de/fdroid/repo?fingerprint=BD08F441BE3C2D84B184230FE7EE4E1EEDEC321E75557548E56E1605C4DE9368

And it did not change from the first time I added the repo (just deleted and re-added the repo).

For releases, we use this `VERSIONID=$(git rev-list HEAD --count main)`.

**Comment by pstorch** [comment]
Found the sha256 fingerprint of the repo in the meanwhile, too.  Question is still: should I change it to be the same as the APK signing key. And the automatically generated repo key is not associated with my name. I don't know if this is of any importance.

**Comment by pstorch** [comment]
> For releases, we use this VERSIONID=$(git rev-list HEAD --count main).

Sounds reasonable. Then we don't need the date based version code.

File: _includes/nightlies.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# [Nightlies](#Nightlies)
+
+Help development and test our nightlies. Add our F-Droid repository https://fdroid.storchp.de/fdroid/repo?fingerprint=BD08F441BE3C2D84B184230FE7EE4E1EEDEC321E75557548E56E1605C4DE9368 to your F-Droid client.
```
Comment (dennisguse) [inline _includes/nightlies.md]: When the link is a <a>, then one can also add the repo when visiting the page by phone

**Review by dennisguse — APPROVED** [review]

File: _includes/nightlies.md (hunk 1 above)
Comment (pstorch) [reply _includes/nightlies.md]: Ok, I'll change it to an a href link.
After reading the docs again and Izzy's tutorial, I think I should create another repo key. I'll fix that first.

Comment (pstorch) [reply _includes/nightlies.md]: The Link is changed.

Still thinking about the repo signing key. The only downside of the current key is the distinguished name which is set to "CN=manjaro, OU=F-Droid", by the fdroid CLI. I don't know if users would check and complain about it. I've never checked distinguished names of keys myself yet.

If I change it, users who already have the repo in their client would probably need to delete and readd it.

Comment (pstorch) [reply _includes/nightlies.md]: Ok, decided to change the signing key before making the repo more public. Now or never. Now it has a meaningful distinguished name.

**Comment by dennisguse** [comment]
So, we can merge this?

**Comment by pstorch** [comment]
I think so, let's merge.

**Pull request merged**